windows.md

```diff
@@ -0,0 +1,29 @@
+Windows Node Support
+
+Windows nodes are partially supported by kubecost as of v1.93.0. Additional support is coming soon!
```

Comment:
capitalize Kubecost

what would you say partial support means? my view is that we either officially support or we don't.